modules/core_project_factory/variables.tf

```
@@ -90,7 +90,12 @@ variable "activate_apis" {
 }
 
 variable "activate_api_identities" {
-  description = "The list of service identities (Google Managed service account for the API) to force-create for the project (e.g. in order to grant additional roles). APIs in this list will automatically be appended to `activate_apis`. Not including the API in this list will follow the default behaviour for identity creation (which is usually when the first resource using the API is created)."
+  description = <<<EOF
```

@umairidris I actually just noticed shouldn't this be 
```suggestion
  description = <<EOF
```

---

D'oh! Good catch. It still isn't passing the docs generation though :(